src/RulesContainer.js

```diff
@@ -103,6 +104,7 @@ export default class RulesContainer {
     if (rule.name) this.map[rule.name] = rule
     if (rule.className && rule.name) this.classes[rule.name] = rule.className
     if (rule.selector) this.map[rule.selector] = rule
+    if (rule.selectorId) this.map[rule.selectorId] = rule
```

Comment:
maybe we can use something one of this (store just rule.selectorId if exists without selector)